Cargo.toml

```diff
@@ -17,23 +17,23 @@ include = ["/src", "LICENSE", "README.md", "CHANGELOG.md"]
 thiserror = "1.0"
 log = "0.4"
 imgui = "^0.12"
-ash = { version = ">=0.34, <=0.37", default-features = false, features = ["debug"] }
+ash = { version = "0.38", default-features = false, features = ["debug"] }
 ultraviolet = "0.9"
 
-gpu-allocator = { version = "0.26", default-features = false, features = ["vulkan"], optional = true }
+#gpu-allocator = { version = "0.26", default-features = false, features = ["vulkan"], optional = true }
+gpu-allocator = { git = "https://github.com/Traverse-Research/gpu-allocator.git", branch = "ash-0.38", default-features = false, features = ["vulkan"], optional = true }
```

Comment:
hi, gpu-allocator 0.27 with ash 0.38 support is available :)

Reply:
I will be afk for 3 weeks starting tomorrow. I'll take a look when I am back but will probably forget so please don't hesitate to ping me if you see no activity here.

Reply:
> I will be afk for 3 weeks starting tomorrow. I'll take a look when I am back but will probably forget so please don't hesitate to ping me if you see no activity here.

This is just a bump because I'd like to use this.